app/services/sign_in/user_loader.rb

```diff
@@ -41,7 +41,9 @@ def validate_account_and_session
     end
 
     def create_mhv_account
-      return unless current_user.loa3?
+      return unless current_user.loa3? && current_user.va_patient?
```

Comment:
Could we throw these conditions into a function on the user model? Like a `should_call_mhv_account_creation?` or something?

Reply:
Even the flipper can go in there I think

Reply:
yeah, i'm working on moving the entire method. Just wanted to confirm the requirements first 😅